lib/internal/tty.js

```diff
@@ -49,9 +49,11 @@ const TERM_ENVS = {
   'dtterm': COLORS_16,
   'gnome': COLORS_16,
   'hurd': COLORS_16,
+  'hyper': COLORS_256,
   'jfbterm': COLORS_16,
   'konsole': COLORS_16,
   'kterm': COLORS_16,
+  'macterm': COLORS_16m,
```

Comment:
We already have support for `macterm`. It also seems the environment variable that is set is `TERM_PROGRAM`, not `TERM`. Something similar probably applies to `hyper` as well.

Reply:
In addition:
```suggestion
  'macterm': COLORS_16,
```

Reply:
hyper sets `TERM_PROGRAM=Hyper`